code-challenges.test.js

```diff
@@ -53,12 +147,38 @@ const studyMinutesWeek2 = {
 
 // b) Create the function that makes the test pass.
 
+const leastToGreatest = (minutes) => {
+  return Object.values(minutes).sort((a, b) => a - b)
+}
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+
 // Pseudo code:
+// 1. Create a function called leastToGreatest. Has one parameter called minutes.
+// 2. Use static Method Object.values(minutes) with our parameter minutes. To call upon the values, not the keys of the object.
```

Comment:
I would add that Object.values will return an array of the values of the object that was passed as an argument